src/wbs-controller_impl.cxx

```diff
@@ -25,7 +25,7 @@ WBSControllerImpl::view_node_selected (
   {
     return;
   }
-
+  
```

Comment:
Remove this diff. Its a whitespace. Better to append the latest commit and force push the changes.